transcription/transcription.go

```diff
@@ -0,0 +1,31 @@
+package transcription
+
+import (
+	"net/smtp"
+	"strings"
+)
+
+// SendEmail authenticates, creates message, and sends email from arguments.
+func SendEmail(username string, password string, host string, port int, to []string, subject string, body string) error {
+	auth := smtp.PlainAuth("", username, password, host)
+
+	// The msg headers should usually include fields such as "From", "To", "Subject", and "Cc".
+	// Sending "Bcc" messages is accomplished by including an email address in
+	// the to parameter but not including it in the msg headers.
+	from := username
+	fromHeader := "From: " + from
+	toHeader := "To: " + strings.Join(to, ", ")
+	subjectHeader := "Subject: " + subject
+	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
```

Comment:
gonna have it return the `[]string` cuz i want to keep all the CRLF termination characters in the same function